products/PurePhone/assets.json

```diff
@@ -2,73 +2,74 @@
         },
         {
             "name": "./fonts/pure/gt_pressura_regular_24.mpf",
-            "output":"image/assets/fonts/gt_pressura_regular_24.mpf"
+            "output":"assets/fonts/gt_pressura_regular_24.mpf"
         },
         {
             "name": "./fonts/pure/gt_pressura_bold_24.mpf",
-            "output":"image/assets/fonts/gt_pressura_bold_24.mpf"
+            "output":"assets/fonts/gt_pressura_bold_24.mpf"
         },
         {
             "name": "./fonts/pure/gt_pressura_light_27.mpf",
-            "output":"image/assets/fonts/gt_pressura_light_27.mpf"
+            "output":"assets/fonts/gt_pressura_light_27.mpf"
         },
         {
             "name": "./fonts/pure/gt_pressura_regular_27.mpf",
-            "output":"image/assets/fonts/gt_pressura_regular_27.mpf"
+            "output":"assets/fonts/gt_pressura_regular_27.mpf"
         },
         {
             "name": "./fonts/pure/gt_pressura_bold_27.mpf",
-            "output":"image/assets/fonts/gt_pressura_bold_27.mpf"
+            "output":"assets/fonts/gt_pressura_bold_27.mpf"
         },
         {
             "name": "./fonts/pure/gt_pressura_bold_32.mpf",
-            "output":"image/assets/fonts/gt_pressura_bold_32.mpf"
+            "output":"assets/fonts/gt_pressura_bold_32.mpf"
         },
         {
             "name": "./fonts/common/gt_pressura_light_30.mpf",
             "ref": "10c74fcb09c2022325767cad735c0183b6f5393a",
-            "output": "image/assets/fonts/gt_pressura_light_30.mpf"
+            "output": "assets/fonts/gt_pressura_light_30.mpf"
         },
         {
             "name": "./fonts/common/gt_pressura_regular_30.mpf",
             "ref": "10c74fcb09c2022325767cad735c0183b6f5393a",
-            "output": "image/assets/fonts/gt_pressura_regular_30.mpf"
+            "output": "assets/fonts/gt_pressura_regular_30.mpf"
         },
         {
             "name": "./fonts/common/gt_pressura_bold_30.mpf",
             "ref": "10c74fcb09c2022325767cad735c0183b6f5393a",
-            "output": "image/assets/fonts/gt_pressura_bold_30.mpf"
+            "output": "assets/fonts/gt_pressura_bold_30.mpf"
         },
         {
             "name": "./fonts/common/gt_pressura_regular_46.mpf",
             "ref": "10c74fcb09c2022325767cad735c0183b6f5393a",
-            "output": "image/assets/fonts/gt_pressura_regular_46.mpf"
+            "output": "assets/fonts/gt_pressura_regular_46.mpf"
         },
         {
             "name": "./fonts/common/gt_pressura_light_46.mpf",
             "ref": "10c74fcb09c2022325767cad735c0183b6f5393a",
-            "output": "image/assets/fonts/gt_pressura_light_46.mpf"
+            "output": "assets/fonts/gt_pressura_light_46.mpf"
         },
         {
             "name": "./fonts/common/gt_pressura_light_90.mpf",
             "ref": "10c74fcb09c2022325767cad735c0183b6f5393a",
-            "output": "image/assets/fonts/gt_pressura_light_90.mpf"
+            "output": "assets/fonts/gt_pressura_light_90.mpf"
```

Comment:
did you ensure yourself that the path is correct? :P

Reply:
yes, but I'll double-check it with the updated assets list for release images.
The best way to verify is to compare build images structure (mounting similar packages with losetup and mount) - then if it's matching, check on hardware :)